common/src/main/java/com/wynntils/features/user/WorldWaypointDistanceFeature.java

```diff
@@ -198,7 +202,7 @@ public void onRenderGuiPost(RenderEvent.Post event) {
 
     private Vec3 worldToScreen(Vector3f delta, Matrix4f projection) {
         Vector4f clipCoords = new Vector4f(delta.x(), delta.y(), delta.z(), 1.0f);
-        clipCoords.transform(projection);
+        projection.transform(clipCoords);
```

Comment:
This inversion of instance and argument looks a bit sus. Is it just that the JOML transform method switches operands compared to Mojangs old version with the same name?

Reply:
https://github.com/Wynntils/Artemis/pull/880#discussion_r1051947100 
> (as I failed to port a singular line of change).
This was it. I have no idea what whichery MC did, or what JOML does, but this is how it is done. And it works.

Reply:
Glad it's not just me then :)